### PR TITLE
a few typo from DSL

### PR DESCRIPTION
**Pull request by yzhao244**
**Please specify parts of this PR update:**

- [ ] Specification
- [ ] Schema
- [ ] Examples
- [ ] Extensions
- [ ] Use Cases
- [ ] Community
- [ ] CTK
- [x] Other

**Discussion or Issue link**:

**What this PR does**:

This PR contains a few typo fix for DSL doc

**Additional information:**

### CODE REVIEW

File: dsl.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ The Serverless Workflow DSL defines several default [task](dsl-reference.md#task
 - [Try](dsl-reference.md#try), used to attempt executing a specified [task](dsl-reference.md#task), and to handle any resulting [errors](dsl-reference.md#error) gracefully, allowing the [workflow](dsl-reference.md#workflow) to continue without interruption.
 - [Wait](dsl-reference.md#wait), used to pause or wait for a specified duration before proceeding to the next task.
 
-To ensure they conform to the DSL, runtimes **should** pass all the feature conformance test scenarii defined in the [ctk](ctk/README.md).
```
Comment (cdavernas) [inline dsl.md:116]: afaik `scenarii` is the correct plural form of `scenario`, no?

**Review by ricardozanini — APPROVED** [review]
Just checked the anchors and they are good and indeed in another file.

Thanks and welcome back :)

File: dsl.md (hunk 1 above)
Comment (ricardozanini) [reply dsl.md:116]: Indeed, but quite ols. I think `scenarios` fits better in this case:

> (nonstandard, rare)

https://www.yourdictionary.com/scenarii

Comment (ricardozanini) [reply dsl.md:116]: Also this guy here gave more background: https://english.stackexchange.com/questions/11775/what-is-the-plural-of-scenario

Sorry to spam to much messages, I was curious about the `scenarii` never knew!! I learned something new today, hahaha

**Comment by ricardozanini** [comment]
@yzhao244 can you please sign your commit?

File: dsl.md (hunk 1 above)
Comment (yzhao244) [reply dsl.md:116]: @cdavernas Actually, you are right lol. Sometime, I use scenarios, but I think scenarii is good. I will change back.

**Comment by yzhao244** [comment]
> @yzhao244 can you please sign your commit?

right, will do that. :)

**Review by cdavernas — APPROVED** [review]
LGTM!

**Comment by cdavernas** [comment]
DCO is still failling. Don't forget to sign-off your commits.

**Comment by yzhao244** [comment]
> DCO is still failling. Don't forget to sign-off your commits.



> DCO is still failling. Don't forget to sign-off your commits.



> DCO is still failling. Don't forget to sign-off your commits.



> DCO is still failling. Don't forget to sign-off your commits.



> DCO is still failling. Don't forget to sign-off your commits.

oh sorry, let me fix my commit. :)

**Comment by yzhao244** [comment]
Fixed my PR, now has one signed off commit. :)

**Pull request merged**